app/controllers/api/episodes_controller.rb

```diff
@@ -3,4 +3,8 @@ class Api::EpisodesController < Api::BaseController
   represent_with Api::EpisodeRepresenter
   filter_resources_by :podcast_id
   find_method :find_by_guid
+
+  def scoped(relation)
+    relation.published.released
```

Comment:
Is there a reason to show only published/released episodes here?  There may be some uses for the API results not being an exact representation of what's in the feed.